docs/user-guide/downward-api/README.md

```diff
@@ -29,4 +29,25 @@ $ kubectl logs dapi-test-pod | grep POD_
 2015-04-30T20:22:18.568024817Z MY_POD_NAME=dapi-test-pod
 2015-04-30T20:22:18.568087688Z MY_POD_NAMESPACE=default
 2015-04-30T20:22:18.568092435Z MY_POD_IP=10.0.1.6
-```
\ No newline at end of file
+```
+
+## Example of environment variables with container resources
+
+Use the [`dapi-container-resources.yaml`](dapi-container-resources.yaml) file to create a Pod
+with a container that consumes the downward API exposing the container's resources.
```

Comment:
Actually if you see the current documentation in index.md and README.md: there are mentions like "..API allows containers to consume information about themselves.." and "..Containers consume information from the downward API .." . and "..Containers consume the downward API using environment variables" .

So I kinda took it from there. IMO, I think the idea is that container is the active unit inside pod and so these statements, I dont disgree with that unless if we want to be very precise and say that applications/processes inside containers consume downward API.

So would you prefer saying "applications or processes inside containers consumes downward API"? Or you have some other thoughts?